src/Utility.cc

```diff
@@ -349,6 +368,25 @@ namespace gz
       return result;
     }
 
+    /////////////////////////////////////////////
+    msgs::SphericalCoordinatesType ConvertCoord(
+      const math::SphericalCoordinates::CoordinateType &_sc)
+    {
+      switch (_sc)
+      {
+        case math::SphericalCoordinates::CoordinateType::ECEF:
+          return msgs::SphericalCoordinatesType::ECEF;
+        case math::SphericalCoordinates::CoordinateType::GLOBAL:
+          return msgs::SphericalCoordinatesType::GLOBAL;
+        case math::SphericalCoordinates::CoordinateType::SPHERICAL:
+          return msgs::SphericalCoordinatesType::SPHERICAL;
+        case math::SphericalCoordinates::CoordinateType::LOCAL:
+          return  msgs::SphericalCoordinatesType::LOCAL;
+        case math::SphericalCoordinates::CoordinateType::LOCAL2:
+          return msgs::SphericalCoordinatesType::LOCAL2;
```

Comment:
This block returns a warning on Windows as we don't return anything if none of the previous conditions are met.